src/Runner.Worker/ExecutionContext.cs

```diff
@@ -510,6 +511,24 @@ public void AddIssue(Issue issue, string logMessage = null)
                 }
 
                 _record.WarningCount++;
+            } 
+            else if (issue.Type == IssueType.Notice) 
+            {
+
+                // tracking line number for each issue in log file
+                // log UI use this to navigate from issue to log
+                if (!string.IsNullOrEmpty(logMessage))
+                {
+                    long logLineNumber = Write(_noticeLogPrefix, logMessage);
```

Comment:
If we prefix with ANSI color codes, should we strip color codes from the log message?

Reply:
interesting offline discussion w/ @luketomlinson 

should we strip color codes from annotations. whether annotations supports special rendering (e.g. html/markdown) will help guide whether we should strip color codes or fix annotations

Reply:
no matter what we do, please make sure `error==warning==notice`